force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -288,8 +368,12 @@ public with sharing class FlowControlLoopsCollections {
     public static Decimal countMoney() {
         // The list of money in the wallet, represented in dollars and cents
         List<Decimal> moneyInWallet = new List<Decimal>{0.50, 10, 3.84, 24.60, 9.08, 50, 4.90}; //DO NOT CHANGE
-
-        return null; // Replace null with the variable you used to store the result
+        Decimal sum = 0.0;
+        for ( Integer i = 0; i < moneyInWallet.size(); i++ ) {
+            sum = sum + moneyInWallet.get(i);
+            if ( sum > 40.0 ) { break; } 
```

Comment:
Try to avoid magic numbers
https://www.inf.unibz.it/~calvanese/teaching/05-06-ip/lecture-notes/uni04/node17.html#:~:text=A%20magic%20number%20is%20a,difficult%20to%20maintain%20and%20update.